proj4/src/main/java/org/osgeo/proj4j/datum/Axes.java

```diff
@@ -0,0 +1,112 @@
+package org.osgeo.proj4j.datum;
+
+import org.osgeo.proj4j.ProjCoordinate;
+
+public final class Axes {
```

Comment:
This, and other classes, must be marked as Serializable (spark tests will fail with serialization errors)